src/components/AccountContainer.js

```diff
@@ -46,15 +46,42 @@ class AccountContainer extends Component {
     }
   }
 
+  componentDidMount(){
+    this.setState({
+      transactions: []
+    })
+    const URL = "https://boiling-brook-94902.herokuapp.com/transactions"
+    fetch(URL)
+      .then(res => res.json())
+      .then(data => {
+        data.map((transaction)=>{
+          var trans =
+              {id: transaction.id,
+              posted_at: transaction.posted_at,
+              description: transaction.description,
+              category: transaction.category,
+              amount: transaction.amount}
+          this.setState({
+            transactions: this.state.transactions.concat(trans)
+          })
+
+        })
+      })
+
+  }
+
   handleChange(event) {
-    // your code here
+    this.setState({
+      searchTerm: event.target.value
+    })
   }
 
```

Comment:
search(){
   return this.state.transactions.filter(transaction => transaction.description.includes(this.state.searchInput) || transaction.category.includes(this.state.searchInput))
}